CMakeLists.txt

```diff
@@ -142,9 +142,14 @@ endif()
 string(REPLACE "include/" "${CMAKE_CURRENT_SOURCE_DIR}/include/" PYBIND11_HEADERS
                "${PYBIND11_HEADERS}")
 
-# Cache variables so pybind11_add_module can be used in parent projects
-set(PYBIND11_INCLUDE_DIR
+# Cache variable so this can be used in parent projects
+set(pybind11_INCLUDE_DIR
```

Comment:
Oh, is the convention of the new CMake system from 2.6.0 to use `pybind11` instead of `PYBIND11`? I hadn't noticed, really.
Or is the an ad-hoc workaround to have 2 versions of the variable?

Reply:
Config mode requires `pybind11_` versions. If you write `find_package(pybind11`, then you have to have `pybind11_*` variables like `pybind11_FOUND`. Otherwise, you'd have to have `find_package(PYBIND11`, which we don't provide (and you can't provide both on Windows, even if you wanted to). The old system had `PYBIND11_*` variables for subdirectory mode only - we have standardized on using the Config mode variable names when they overlap rather than having to change this based on which mode you used. This is provided for backward compatibility only.

Internally we were still using `PYBIND11_INCLUDE_DIR` and then setting `pybind11_INCLUDE_DIR` at the end - it made more sense to use the proper variable everywhere and just provide `PYBIND11_INCLUDE_DIR` in subdirectory mode.

Reply:
Right, OK, that makes sense, thanks! :-) I was just thinking that I was now using `PYBIND11_FINDPYTON=ON` etc...

Reply:
Yes, variables you set are `PYBIND11_*`. Variables you read are `pybind11_*`.

Reply:
OK, another convention I just learned about. Thanks! :-)